server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -15,6 +15,12 @@ CREATE TABLE "ORGANIZATIONS" (
 CREATE UNIQUE INDEX "PK_ORGANIZATIONS" ON "ORGANIZATIONS" ("UUID");
 CREATE UNIQUE INDEX "ORGANIZATION_KEY" ON "ORGANIZATIONS" ("KEE");
 
+CREATE TABLE "ORGANIZATION_MEMBERS" (
+  "UUID" VARCHAR(40) NOT NULL PRIMARY KEY,
```

Comment:
why a UUID on an association table?
the primary key should be composite of ORGANIZATION_UUID and USER_ID. In addition, it would enforce uniqueness of information.